sentence_transformers/sparse_encoder/models/MLMTransformer.py

```diff
@@ -106,13 +106,9 @@ def forward(self, features: dict[str, torch.Tensor]) -> dict[str, torch.Tensor]:
             for key, value in features.items()
             if key in ["input_ids", "attention_mask", "token_type_ids", "inputs_embeds"]
         }
-        # Get MLM outputs
-        mlm_outputs = self.auto_model(**trans_features)
-
-        # Get the MLM head logits (shape: batch_size, seq_length, vocab_size)
-        mlm_logits = mlm_outputs.logits
-
-        features["token_embeddings"] = mlm_logits
+        features["token_embeddings"] = self.auto_model(
+            **trans_features, output_hidden_states=False, output_attentions=False
```

Comment:
I don't think we should specify the `output_hidden_states` and `output_attentions`, these are False by default unless the user specifies otherwise in the `transformers` `AutoConfig`, and then I don't think we need to manually specify it ourselves.